### PR TITLE
python3Packages.pyjnius: init at 1.4.0

### LINKED ISSUE #169073: Pipenv: No module named 'jnius'
### Describe the bug
`Traceback (most recent call last):
  File "/nix/store/z4sd700l05shq1z72g21w0fav2xs955d-pipenv-2022.1.8/lib/python3.9/site-packages/pipenv/vendor/platformdirs/android.py", line 83, in _android_folder
    from jnius import autoclass  # noqa: SC200
ModuleNotFoundError: No module named 'jnius'`

### Expected behavior
The error is belong in pipenv using jnius on Android. 

### Metadata
 - system: `"aarch64-linux"`
 - host os: `Linux 3.10.84-perf-g39a29a8`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.3.16`
 - channels(nix-on-droid): `"home-manager-21.11, nix-on-droid-21.11, nixpkgs-21.11.337090.0a862f558d4"`
 - nixpkgs: `/data/data/com.termux.nix/files/home/.nix-defexpr/channels/nixpkgs`

### PR DESCRIPTION
**Pull request by ifurther**
###### Description of changes
Add [pyjnius](https://github.com/kivy/pyjnius) , and it didn't use pytest. It is the dependent package for pipenv in Android. Resolved #169073

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pyjnius/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+buildPythonPackage rec {
+  pname = "pyjnius";
+  version = "1.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "8bc1a1b06fb11df8dd8b8d56f5ecceab614d4ba70cf559c64ae2f146423d53ce";
+  };
+  propagatedBuildInputs = [
+  six
```
Comment (legendofmiracles) [inline pkgs/development/python-modules/pyjnius/default.nix:20]:
```suggestion
    six
```

File: pkgs/development/python-modules/pyjnius/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  ];
+
+  nativeBuildInputs = [ jdk cython ];
+
+  pythonImportsCheck = [ "jnius" ];
+
+  meta = with lib; {
+    description = "A Python module to access Java classes as Python classes using the Java Native Interface (JNI).";
```
Comment (legendofmiracles) [inline pkgs/development/python-modules/pyjnius/default.nix:28]:
```suggestion
    description = "A Python module to access Java classes as Python classes using the Java Native Interface (JNI)";
```

File: pkgs/development/python-modules/pyjnius/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, cython
+, fetchPypi
+, isPy3k
+, jdk
+, lib
+, six
+, pythonOlder
```
Comment (legendofmiracles) [inline pkgs/development/python-modules/pyjnius/default.nix:8]: Where are isPy3k and pythonOlder used? I believe they can be removed

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -5418,6 +5418,12 @@
     githubId = 1550265;
     name = "Dominic Steinitz";
   };
+  ifurther = {
+    email = "55025025+ifurther@users.noreply.github.com";
+    github = "ifurther";
+    githubId = 55025025;
+    name = "Feather Lin";
+  };
```
Comment (legendofmiracles) [inline maintainers/maintainer-list.nix:5426]: Different commit: `maintainers: add ifurther`

File: pkgs/development/python-modules/pyjnius/default.nix (hunk 3 above)
Comment (ifurther) [reply pkgs/development/python-modules/pyjnius/default.nix:8]: OK, I removed

**Comment by onny** [comment]
Could you please squash this commit into an existing one? [Merge remote-tracking branch 'upstream/master' into add-python-pyjnius](https://github.com/NixOS/nixpkgs/pull/177170/commits/10f650c49de5f2cdb4d908357c10d33d4b3cc070)

**Comment by onny** [comment]
Thank you!

**Pull request merged**